ScoutSuite/__init__.py

```diff
@@ -7,12 +7,14 @@
 EXCEPTIONS = 4242
 HTMLREPORT = 424242
 AWSRULESET = 42424242
+ERRORSLOG = 4242424242
```

Comment:
I feel like the 42 thing won't scale very well. Maybe we should start using legit error codes hehe.

Reply:
Well considering it's absurde origin (https://en.wikipedia.org/wiki/The_Hitchhiker%27s_Guide_to_the_Galaxy#The_Hitchhiker's_Guide_to_the_Galaxy)...

Reply:
:+1: for legit error codes pl0x

Reply:
I am aware of the origin, great book, still think we should use legit error codes 😂

Reply:
Oh I completely agree, it's ridiculous. I'll do a bit of refactoring on this because it's not actually an error code / actually useful.